library/modules/bigip_snat_pool.py

```diff
@@ -165,6 +194,14 @@ def members(self):
             result.update([address])
         return list(result)
 
+    @property
+    def description(self):
```

Comment:
this one is ok as you give the user an option to specify 'none' instead of empty string i guess this can stay

Reply:
Ok, I need to understand better where to add properties and where not to.